core-web/libs/edit-content/src/lib/fields/dot-edit-content-file-field/components/dot-form-file-editor/dot-form-file-editor.component.html

```diff
@@ -0,0 +1,59 @@
+<form (ngSubmit)="onSubmit()" [formGroup]="form" class="editor-mode__form">
+    @if (store.allowFileNameEdit()) {
+        <div class="editor-mode__input-container">
+            <label class="p-label-input-required" data-testId="editor-label" for="file-name">
+                {{ 'dot.file.field.action.create.new.file.label' | dm }}
+            </label>
+            <input
+                id="file-name"
+                autocomplete="off"
+                formControlName="name"
+                pInputText
+                placeholder="Ex. template.html"
+                type="text"
+                data-testId="editor-file-name" />
+            <div class="error-message">
+                @let error = store.error();
+                @if (error) {
+                    <small class="p-invalid" data-testId="error-msg">
+                        {{ error | dm: [store.allowFiles()] }}
+                    </small>
+                } @else {
+                    <dot-field-validation-message
+                        [field]="nameField"
+                        [patternErrorMessage]="'dot.file.field.error.type.file.not.extension'"
+                        data-testId="error-message" />
+                }
+            </div>
+        </div>
+    }
+    <div class="file-field__editor-container">
+        <ngx-monaco-editor
+            [class.file-field__code-editor--disabled]="form.disabled"
```

Comment:
You can disabled the Monaco Editor with the config, maybe is better. I think Humberto did something like in the portlet.
```
editorOptions = {
    readOnly: true,
   ...
};
```

Reply:
That line is only for styling, but I have also disabled the editor in the code

![Screenshot 2024-10-24 at 8 48 09 AM](https://github.com/user-attachments/assets/286d8c33-154c-408a-9b63-3f5d22924048)